### ISSUE TITLE
The widget (admin area)

### ISSUE THREAD

**Issue by emiluzelac** [quotepro/quotepro-insurance-widget#14]
The widget ID field is made the same way as in #13. What I also noticed is that switching to Español nothing is happening, so not sure if this is still work in progress or...

**Comment by b2k** [comment]
There has been no translation done on the plugin. What that does is to change the target url.

**Comment by emiluzelac** [comment]
Cool 👍

**Comment by emiluzelac** [comment]
In that case maybe rewording to Language target or site, just to avoid
confusion.
